src/utils/events/index.ts

```diff
@@ -253,3 +253,26 @@ export const getActionValues = async (
     }
   }
 };
+
+export const getAnnotation = async (
+  agent: Address,
+  transactionHash: string,
+  colonyClient: ColonyClient,
+) => {
+  /*
+   * @TODO For some reason types were not autogenerated in colonyJS for this filter
+   * I need, at some point, to investigate what's up
+   */
+  // @ts-ignore
+  const filter = colonyClient.filters.Annotation(null, null);
```

Comment:
```suggestion
  const filter = colonyClient.filters.Annotation();
```
Would also works fine

Reply:
You're working from an old version here as this was changed to pass in the `agent` and the `transactionHash`.

Reply:
See 173c510f10e2c371f72a6f2c11cc0714effd9f7b